2.0/parser/filing_parser.py

```
@@ -0,0 +1,1799 @@
+            logging.info(f"Concatenating the DF's of {len(parsed_pdfs)} PDF files for the current {self.type} filing.")
+            return pd.concat(parsed_pdfs, ignore_index=True)
+        
+        logging.info(f'Unable to find or parse any PDFs for the current {self.type} filing.')
+        return pd.DataFrame()
+
+    def is_text_page_number(self, question_text):
+        '''
+        Support method for find_section_with_toc, attempt to determine if the given text is simple a page number (duplicate link in my observations).
+        '''
+        # Check argument
+        if type(question_text) != str:
```

Ruff (0.11.9)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)